tests/e2e/workloads/amq/test_rgw_kafka_notifications.py

```diff
@@ -0,0 +1,185 @@
+import boto3
+import logging
+import pytest
+import re
+import tempfile
+
+from subprocess import CalledProcessError, run
+
+from ocs_ci.framework.testlib import E2ETest, tier1, vsphere_platform_required, bugzilla
+from ocs_ci.helpers.helpers import default_storage_class
+from ocs_ci.ocs.amq import AMQ
+from ocs_ci.ocs.bucket_utils import retrieve_verification_mode
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.exceptions import CommandFailed
+from ocs_ci.ocs.ocp import OCP
+from ocs_ci.ocs.resources.ocs import OCS
+from ocs_ci.ocs.resources.objectbucket import OBC
+from ocs_ci.ocs.resources.rgw import RGW
+from ocs_ci.ocs.resources.pod import get_pod_logs
+from ocs_ci.ocs.utils import get_pod_name_by_pattern
+from ocs_ci.utility import templating
+from ocs_ci.utility.utils import run_cmd
+
+
+log = logging.getLogger(__name__)
+
+
+def clone_notify():
+    """
+    Returns notify path
```

Comment:
can you please add more details about this function - like why https://github.com/shonpaz123/notify/ is needed and how it's used?....